salt/modules/netmiko_mod.py

```diff
@@ -501,7 +502,7 @@ def send_config(config_file=None,
                 attrs=None,
                 context=None,
                 defaults=None,
-                skip_verify=False,
+                skip_verify=True,
```

Comment:
Why is this default changing? That at least needs to be documented.

Reply:
Indeed, it shouldn't. Reverted.